src/Lykke.Service.OperationsHistory.Mongo/OperationsHistoryRepository.cs

```diff
@@ -49,25 +49,24 @@ public class OperationsHistoryRepository : IOperationsHistoryRepository
             return (await _collection.Find(x => x.ClientId == clientId && x.Id == id).ToListAsync()).Single();
         }
 
-        public async Task<IEnumerable<OperationsHistoryEntity>> GetByClientIdAsync(
-            string clientId,
+        public async Task<IEnumerable<OperationsHistoryEntity>> GetByClientIdAsync(string clientId,
             string walletId,
-            HistoryOperationType? operationType,
+            HistoryOperationType[] operationTypes,
             string assetId,
             string assetPairId,
-            int take, 
+            int take,
             int skip)
         {
             var result = new List<OperationsHistoryEntity>();
 
-            var queryByOperationType = operationType.HasValue;
-            var type = operationType ?? HistoryOperationType.Trade;
+            var queryByOperationType = operationTypes != null && operationTypes.Any();
+            var types = operationTypes != null && operationTypes.Any() ? operationTypes : new[] {HistoryOperationType.Trade};

```

Comment:
why do you need `new[] {HistoryOperationType.Trade}` ? the code can be simplified.